src/Display/Modal/ModalStyle.ts

```diff
@@ -193,3 +131,87 @@ export const ModalFooter = styled.footer<{ isChildrenInMultiLines: boolean }>`
     `;
   }}
 `;
+
+export const ModalContentArea = styled.div<InnerProps>`
+  position: relative;
+  background: ${({ hideContentArea }) =>
+    hideContentArea ? 'transparent' : `${Greyscale.white}`};
+  margin: 120px auto;
+  max-width: 95vw;
+  outline: none;
+
+  ${({ size }) => {
+    switch (size) {
+      case 's':
+        return `
+          width: ${SIZES.s}px;
+        `;
+      case 'm':
+        return `
+          width: ${SIZES.m}px;
+        `;
+      case 'l':
+        return `
+          width: ${SIZES.l}px;
+        `;
+      case 'xl':
+        return `
+          width: ${SIZES.xl}px;
+        `;
+      default:
+        return `
+          width: ${SIZES.default}px;
+        `;
+    }
+  }}
+
+  ${({ centering }) => {
+    if (centering) {
+      return `
+        display: flex;
+        flex-direction: column;
+        max-height: 85vh;
+      `;
+    }
+  }}
+
+  ${({ fullscreen }) => {
+    if (fullscreen) {
+      return `
+        display: flex;
+        flex-direction: column;
```

Comment:
Why `flex`?

Reply:
to make footer be sticky to bottom
```
      ${ModalFooter} {
          margin-top: auto;
        }
```
![image](https://user-images.githubusercontent.com/13249748/114162550-c91b1100-995b-11eb-8511-3acbbf2913eb.png)